woocommerce/woocommerce-gutenberg-products-block
# Add basic product archive block template

For WooCommerce merchants to be able to fully utilize block themes, we need the most important WooCommerce core PHP templates to be available as block templates.


This PR adds a block template version of the [archive-product.php template](https://github.com/woocommerce/woocommerce/blob/trunk/plugins/woocommerce/templates/archive-product.php).

Closes https://github.com/woocommerce/woocommerce-gutenberg-products-block/issues/5033

|Large viewports|Small viewports|
|-|-|
|<img width="1065" alt="Screenshot 2021-11-02 at 12 57 01" src="https://user-images.githubusercontent.com/1562646/139842016-ac00b413-1f21-4fed-aa6c-46958526626c.png">|<img width="499" alt="Screenshot 2021-11-02 at 12 57 20" src="https://user-images.githubusercontent.com/1562646/139842037-be7be19d-836f-4b18-8147-752d649fd4d9.png">|

### Manual Testing

How to test the changes in this Pull Request:

**Prerequisite**: Please ensure you have a Block Template Theme activated such as TT1 and also the Gutenberg Plugin installed.

1. Load the Site Editor (Appearance > Editor) and select the `Product Archive Page` template from the `General templates` section.
2. Confirm you can load the template and it looks as expected in large/small viewports as per the screenshots above.
3. Confirm the behavior on the Frontend on `/shop` is identical to the behavior on `/shop` without this PR applied.

Once you have followed the above steps, add the same template files to your `theme-dir/block-templates` and change the file contents so you can differentiate between the Woo Block template and the Theme template. Repeat the above steps starting from Step 2 and confirm the Theme template is preferred over the Woo Blocks template

### Changelog

> FSE: Add basic product archive block template.
